docs/en/commands/template.md

```diff
@@ -1,114 +1,115 @@
-# Template <Name of Command>
+# Template
```

Comment:
we probably don't want to output the template as markdown? the template is just for people who will be writing the code, right?

Reply:
Yeah, I can take that out. I just wanted the PR to show the actual product of parsing.